### PR TITLE
log CI tests

### PR DESCRIPTION
**Pull request by pohly**
#### What type of PR is this?

/kind cleanup

#### What this PR does / why we need it:

We have to fork the kind E2E script after all because advanced settings like controlling kubelet flags are not supported. To avoid a complex API for that script, users are meant to fork it.

#### Does this PR introduce a user-facing change?
```release-note
NONE
```

### CODE REVIEW

File: test/integration/logs/e2e-k8s.sh [hunk 1]
```diff
@@ -0,0 +1,301 @@
...
+
+  # debug cluster version
+  kubectl version
+
+  # Patch kube-proxy to set the verbosity level
+  kubectl patch -n kube-system daemonset/kube-proxy \
+    --type='json' -p='[{"op": "add", "path": "/spec/template/spec/containers/0/command/-", "value": "--v='"${KIND_CLUSTER_LOG_LEVEL}"'" }]'
+}
+
+# run e2es with ginkgo-e2e.sh
```
Comment (mengjiao-liu) [inline test/integration/logs/e2e-k8s.sh:209]: From the script, this should be an e2e test, but why put this script in the directory `test/integration/`?

**Comment by aojea** [comment]
the forked scripts of kind use to live in test-infra , per example, https://github.com/kubernetes/test-infra/blob/master/experiment/kind-multizone-e2e.sh 

I think that is better to not add them to k/k

**Comment by pohly** [comment]
Having the script in k/k has the advantage that the Prow job doesn't need to fetch it.

I picked `test/integration/logs` because that is already owned by the right group of people. We can also use `test/e2e/instrumentation/logging`.

**Pull request closed without merging**